exercises/wordy/canonical-data.json

```diff
@@ -134,6 +134,38 @@
         "question": "Who is the President of the United States?"
       },
       "expected": false
+    },
+    {
+      "description": "reject two operations in a row",
+      "property": "answer",
+      "input": {
+        "question": "What is 1 plus plus 2?"
```

Comment:
let's ignore the fact that in some languages, `1 + + 2` is a valid expression that evaluates to `3`. The reason for the validity is that the second `+` sign is a unary plus. We are probably assuming in these word problems that the `plus` refers only to the binary operation.